packages/backend/src/createRedirect.ts

```diff
@@ -1,6 +1,6 @@
 import { errorThrower, parsePublishableKey } from './util/shared';
 
-const buildUrl = (_baseUrl: string | URL, _targetUrl: string | URL, _returnBackUrl?: string | URL) => {
+const buildUrl = (_baseUrl: string | URL, _targetUrl: string | URL, _returnBackUrl?: string | URL | null) => {
```

Comment:
❓  why did we prefix all the params with `_`? I thought we prefix a variable with an underscore to mark it as ignored which is not the case here. Should we rename the params in another PR?